README.md

```diff
@@ -46,17 +46,32 @@ When using such data management platform to overcome the block disk device limit
 - capacity management and monitoring
 - resizing and optimizing layouts related to capacity management
 - decommissioning the devices in secure way
+  - by default every resource created by DiskoBlock has a finalizer, so deletion will be blocked until coresponding DiskConfig is exists
+  - by default every additional disk has owner reference to the first disk ever created for pod, Deletion of first PVC terminates all other
 
 At the current stage, Discoblocks is leveraging the available hyperscaler CSI (Container Storage Interface) within the Kubernetes cluster to:
 - introduce a CRD (Custom Resource Definition) per workload with
+  - StorageClass name
   - capacity
   - mount path within the Pod 
   - nodeSelector
   - podSelector
-  - upscale policy 
+  - access modes: Access mode of PersistentVolume
+  - availability mode:
+    - ReadWriteOnce: New disk for each pod, including pod restart
+    - ReadWriteSame: All pod gets the same volume on the same node
+    - ReadWriteDaemon: DaemonSet pods gets the same volume per node
```

Comment:
What does "same volume per node mean"? If we have EBS disks does this mean that a daemonset will have 1 EBS volume which will be mounted on every node?